modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -212,6 +214,10 @@ public void processOpts() {
             setModelPropertyNaming((String) additionalProperties.get(CodegenConstants.MODEL_PROPERTY_NAMING));
         }
 
+        if (additionalProperties.containsKey(CodegenConstants.PARAM_PROPERTY_NAMING)) {
```

Comment:
I think it should be made a part of cli options - which would automatically add it to documentation. Otherwise nobody will ever know about it.

See how other options are added in the constructor: https://github.com/OpenAPITools/openapi-generator/blob/eb28082c6187d638b590c0dc75adce78a1153444/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java#L176

Reply:
thanks, I will make that change.